src/aat/resources/features/F-103/S-600.td.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "must return 400 error response for mal-formatted user id list",
+  "_guid_": "S-603",
+  "productName": "CCD Data Store",
+  "operationName": "Get Case-Assigned Users and Roles",
+  "_extends_": "F-103_Test_Data_Base",
+
+"specs": [
+    "Richard - who can create a case",
+    "Dil - with an active profile",
+    "Steve - with an active profile",
+    "Jamal -  who is a privileged user with permissions to access the case assignments of other users",
+    "by Jamal to assign Dil a few case roles to access C1",
+    "by Jamal to assign Dil a few case roles to access C2",
+    "by Jamal to assign Dil a few case roles to access C3",
+    "by Jamal to assign Steve a few case roles to access C1",
+    "by Jamal to assign Steve a few case roles to access C2",
+    "by Jamal to assign Steve a few case roles to access C3",
+    "is made by Jamal with Case IDs of C1, C2 & C3 & no User ID"
```

Comment:
are you sure you will have all these specs here? they look like specs to go to the respective api call test data.

Reply:
Please check other test data files to confirm you have all and only the necessary specs - no more, no less. Less than enough fails test, more than enough makes the maintenance quite confusing later.

Reply:
removed these from main spec